runners/direct-java/src/main/java/org/apache/beam/runners/direct/portable/ReferenceRunner.java

```diff
@@ -168,6 +170,7 @@ public void execute() throws Exception {
             .setWorkerId("foo")
             .setResourceLimits(Resources.getDefaultInstance())
             .build();
+    JobInfo jobInfo = JobInfo.create("id", "reference", "retrieval-token", options);
```

Comment:
Should the reference runner allow for multiple (concurrent?) jobs? If so, we may need to actually generate unique job ids here.

Reply:
Good point. I think it's a bit outside the scope of this PR, but I wrote a bug and added a TODO so I can look into it later.

Since this PR right now is mainly for trying to get #6073 working I don't really want to spend too long working on the JobInfo yet (although I will almost certainly need to get to it at some point).